resources/scss/componets/01_basics.scss

```diff
@@ -0,0 +1,131 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+// colors variable start
+
+$black: #000000;
+$white: #ffffff;
+$theme-color: #F51963;
+$theme-color-20: rgba(245, 25, 99, 0.2);
+$border-color: lightgray;
+$linear-gradient-theme: linear-gradient(45deg, #F51963 0%, #FBAE20 100%);
+$background-theme-color: #FAF9FF;
+$water-color: rgba(255, 255, 255, 0.1);
+
+// colors variable end
+
+// global start
+
+.background-theme-color{
+    background-color: $background-theme-color !important;
+}
+
+.btn-theme {
+    color: $white;
+    display: inline-block;
+    text-decoration: none;
+    padding: 12px 40px;
+    background: $theme-color;
+    border-radius: 30px;
+    position: relative;
+    z-index: 3;
+    margin-left: 10px;
+    margin-right: 10px;
+    transition: all .4s;
+    cursor: pointer;
+    outline: none !important;
+    border: 1px solid $theme-color;
+
+    &:after{
+        content: '';
+        width: 0;
+        height: 100%;
+        background: $white;
+        border-radius: 30px;
+        position: absolute;
+        left: 50%;
+        top: 0;
+        z-index: -1;
+        opacity: .5;
+        transition: all .4s;
+    }
+
+    &:hover{
+        background-color: transparent;
+        border: 1px solid $white;
+        color: $theme-color;
+
+        &:after{
+            left: 0;
+            opacity: 1;
+            width: 100%;
+            color: $theme-color;
+        }
+
+    }
+
+}
```

Comment:
Move this scss into **buttons.scss** and remove this from this file please

Reply:
i move this code into buttons.scss